step.yml

```diff
@@ -47,9 +47,11 @@ is_skippable: false
 deps:
   brew:
   - name: go
```

Comment:
A sidenote: go does not need to be added as a dependency, based on the guide: path::/Users/lpusok/Develop/go/src/github.com/bitrise-steplib/steps-project-scanner: